### PR TITLE
Change url for mac compability

### PR DESCRIPTION
**Pull request by infinitely-irrational**

### CODE REVIEW

File: client/src/components/Home.js [hunk 1]
```diff
@@ -1,43 +1,44 @@
-import { useEffect, useState } from 'react';
-import styled from 'styled-components';
+import { useEffect, useState } from "react";
+import styled from "styled-components";
```
Comment (aziz512) [inline client/src/components/Home.js]: what code formatter are you using? single quotes are usually the standard in modern js.
We should all be using the same one or we'll keep having changes like this in our PRs. I use Prettier in vscode.

Comment (infinitely-irrational) [reply client/src/components/Home.js]: I have prettier. I think with my settings it uses double quotes for imports and single quotes otherwise.

Comment (aziz512) [reply client/src/components/Home.js]: Can we change that?

Comment (aziz512) [reply client/src/components/Home.js]: create-react-app boilerplate uses single quotes everywhere as well

Comment (infinitely-irrational) [reply client/src/components/Home.js]: Yup, working on it.

Comment (infinitely-irrational) [reply client/src/components/Home.js]: @aziz512 Fixed.

**Review by aziz512 — APPROVED** [review]
🦾

**Pull request merged**